server/src/main/java/com/epam/aidial/core/server/controller/DeleteFileController.java

```diff
@@ -2,31 +2,19 @@
 
 import com.epam.aidial.core.server.Proxy;
 import com.epam.aidial.core.server.ProxyContext;
-import com.epam.aidial.core.server.service.InvitationService;
-import com.epam.aidial.core.server.service.ShareService;
 import com.epam.aidial.core.server.util.ProxyUtil;
 import com.epam.aidial.core.storage.http.HttpStatus;
 import com.epam.aidial.core.storage.resource.ResourceDescriptor;
-import com.epam.aidial.core.storage.service.LockService;
-import com.epam.aidial.core.storage.service.ResourceService;
 import com.epam.aidial.core.storage.util.EtagHeader;
 import io.vertx.core.Future;
 import lombok.extern.slf4j.Slf4j;
 
 @Slf4j
 public class DeleteFileController extends AccessControlBaseController {
```

Comment:
Seems like redundant to me. We can call resource controller to delete it.

Reply:
Right I have the same feelings

Reply:
deleted